.github/workflows/test-application.yaml

```
@@ -0,0 +1,89 @@
+name: Test application
+
+on:
+    pull_request:
+    push:
+        branches:
+            - 'master'
+            - '[0-9]+.x'
+            - '[0-9]+.[0-9]+'
+            - '[0-9]+.[0-9]+.x'
+
+jobs:
+    test:
+        name: 'PHP ${{ matrix.php-version }}, Database ${{ matrix.db }}, Dependencies ${{ matrix.dependencies }}'
```

```suggestion
        name: 'PHP ${{ matrix.php-version }}, Database ${{ matrix.db }} ${{ matrix.dependencies }}'
```

I'd like to shorten this as it is not readable even on a large screen